src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -2814,7 +2820,8 @@ private DestructorDeclarationSyntax ParseDestructorDeclaration(string typeName,
             out BlockSyntax blockBody,
             out ArrowExpressionClauseSyntax expressionBody,
             out SyntaxToken semicolon,
-            bool parseSemicolonAfterBlock = true)
+            bool parseSemicolonAfterBlock = true,
+            bool isAccessorBody = false)
```

Comment:
A leftover from the development cycle. Reverted back to original signature and code.